apps/src/lib/script-editor/LevelTokenDetails.jsx

```diff
@@ -183,6 +186,13 @@ const LevelTokenDetails = React.createClass({
             />
           </div>
         )}
+        {/* We don't currently support editing progression names here, but do
+          * show the current progression if we have one. */}
+        {this.props.level.progression &&
+          <div style={styles.progression}>
+            Progression Name: {this.props.level.progression}
+          </div>
+        }
```

Comment:
If I move the levels around, it does look like they lose their progressions. This seems somewhat unfortunate, but given that it doesnt really sound like the beta script editor is really fully supported, I'm rather reluctant to invest in fixing this. Plan to take a quick look and see if it's easy, otherwise I'll likely push to punt.